src/servers/CMakeLists.txt

```diff
@@ -129,8 +141,13 @@ if(${TRTIS_ENABLE_TRACING})
   add_library(
     tracing-library EXCLUDE_FROM_ALL OBJECT
     ${TRACING_SRCS} ${TRACING_HDRS}
+    ${CUDA_OBJS}
   )
 
+  if(${TRTIS_ENABLE_GPU})
+    target_include_directories(tracing-library PRIVATE ${CUDA_INCLUDE_DIRS})
```

Comment:
Why does the tracing library need cuda?

Reply:
The tracing needs CUDA since it includes trtserver (which uses cuda when TRTIS_ENABLE_GPU is ON)